### PR TITLE
RFC: Remove various bits which support non-GPLv3 builds

### PR DESCRIPTION
**Pull request by kergoth**
Signed-off-by: Christopher Larson <chris_larson@mentor.com>

### CODE REVIEW

**Comment by kergoth** [comment]
Not sure on this one. I don't think we actively support non-gplv3 builds anymore, though.

**Comment by abelal** [comment]
We should probably add @chombourger to this discussion as well. Officially, we do not support GPLv3 excluded builds with the product releases anymore however we know that some of the customers are interested. We should probably remove this from here but create a separate layer with these changes that the services teams could work with and maintain. This would at least give them a starting point.

**Comment by chombourger** [comment]
It's definitely alright for Flex 13. Services team should create a layer to support non-GPLv3 builds that they would only deliver to selected customers/projects

File: conf/local.conf.sample [hunk 1]
```diff
@@ -96,21 +96,6 @@ BB_NUMBER_THREADS ?= "${@int(oe.utils.cpu_count() * 1.5)}"
 # How many jobs 'make' should run in parallel when compiling
 PARALLEL_MAKE ?= "-j ${@oe.utils.cpu_count() * 2}"
 
-# Uncomment to exclude GPLv3 software from the build
-#INCOMPATIBLE_LICENSE = "GPL-3.0* LGPL-3.0*"
```
Comment (Noor-Ahsan) [inline conf/local.conf.sample:100]: @chombourger it was decided that the option will remain there in local.conf to build with non GPLV3

Comment (kergoth) [reply conf/local.conf.sample:100]: @Noor-Ahsan Ah, if that's the case then this PR should be dropped, as it won't be usable without the rest.

**Pull request closed without merging**